### DISCUSSION TITLE
tailwind ui figma kit now leads to a 404

### DISCUSSION THREAD

**Discussion by bstro** [tailwindlabs/tailwindcss#5226] [Help] [answered]
This might be slightly OT, but I can't seem to find the Tailwind UI figma kit anymore. The top result from a google search leads to this https://www.figma.com/community/file/958381589801778722 which currently 404s. I also can't find any links to this file from tailwindui.com. Currently trying to vet the tooling before making a recommendation to my team.

**Comment by simonswiss** [comment, marked as answer]
Hey!

We had to make the really hard decision to **discontinue the Tailwind UI Figma UI Kit**. They were just an absolutely enormous amount of work to maintain and very few people were using them.

Making this tough call allows us to spend more time on the code snippets, which is where we think we can provide the most value.

If you're a Tailwind UI customer, we can enable the current Figma assets for your account (send an email to support@tailwindui.com if you want that). However, these won't include the new Ecommerce assets and won't be updated anymore.

Thank you for your understanding 🙏
